test-data/unit/check-38.test

```diff
@@ -106,3 +106,71 @@ def g(x: int): ...
     /
     0  # type: ignore
 )  # type: ignore  # E: unused 'type: ignore' comment
+
+-- Unskip the following PEP 570 tests once Travis supports 3.8.0a4 (or greater).
+
+[case testPEP570ArgTypesMissing-skip]
+# flags: --disallow-untyped-defs
+def f(arg, /) -> None: ...  # E: Function is missing a type annotation for one or more arguments
+
+[case testPEP570ArgTypesBadDefault-skip]
+def f(arg: int = "ERROR", /) -> None: ...  # E: Incompatible default for argument "arg" (default has type "str", argument has type "int")
+
+[case testPEP570ArgTypesDefault-skip]
+def f(arg: int = 0, /) -> None:
+    reveal_type(arg)  # E: Revealed type is 'builtins.int'
+
+[case testPEP570ArgTypesRequired-skip]
+def f(arg: int, /) -> None:
+    reveal_type(arg)  # E: Revealed type is 'builtins.int'
+
+[case testPEP570Required-skip]
+def f(arg: int, /) -> None: ...  # N: "f" defined here
+f(1)
+f("ERROR")  # E: Argument 1 to "f" has incompatible type "str"; expected "int"
+f(arg=1)  # E: Unexpected keyword argument "arg" for "f"
+f(arg="ERROR")  # E: Unexpected keyword argument "arg" for "f"
+
+[case testPEP570Default-skip]
+def f(arg: int = 0, /) -> None: ...  # N: "f" defined here
+f()
+f(1)
+f("ERROR")  # E: Argument 1 to "f" has incompatible type "str"; expected "int"
+f(arg=1)  # E: Unexpected keyword argument "arg" for "f"
+f(arg="ERROR")  # E: Unexpected keyword argument "arg" for "f"
+
+[case testPEP570Signatures1-skip]
+def f(p1: bytes, p2: float, /, p_or_kw: int, *, kw: str) -> None:
```

Comment:
Try also calling `f` with valid and invalid arguments. For example, make sure `p1` is not a valid keyword argument but `p_or_kw` is both a valid positional and keyword argument, and `kw` must be a keyword argument.